src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -85,7 +85,7 @@ exports[`Button Component renders button with icon right correctly 1`] = `
 exports[`Button Component renders circle button correctly 1`] = `
 <DocumentFragment>
   <button
-    class="ant-btn css-dev-only-do-not-override-1vr7spz ant-btn-circle ant-btn-primary button"
+    class="ant-btn css-dev-only-do-not-override-1vr7spz ant-btn-circle ant-btn-primary button buttonMdIconOnly"
```

Comment:
This is out of scope. It should be fixed in a specific merge request

Reply:
It is a remnant of the PR on Button, where I had forgotten to update the snapshots (we should configure the pipeline so that it blocks the merge operation if it fails). To me, it seems a little superfluous to open an ad-hoc PR